terraform/app.tf

```diff
@@ -4,6 +4,8 @@ locals {
       DOCKER_REGISTRY_SERVER_URL            = "https://ghcr.io",
       DATABASE_URL                          = "postgres://postgres@${local.postgres_server_name}.postgres.database.azure.com:5432"
       DATABASE_PASSWORD                     = local.infrastructure_secrets.POSTGRES_ADMIN_PASSWORD
+      HOSTING_DOMAIN                        = var.domain != null ? "https://${var.domain}" : "https://${local.aytq_web_app_name}.azurewebsites.net"
```

Comment:
where are we setting var.domain?

Reply:
It's being set [here](https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/5af6a2d00a3621b12d6e135a781e91a11ef35ea1/terraform/variables.tf#L124), but not in use yet